src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -614,4 +622,12 @@ private void rotateLeftClicked()
         rotation = (rotation + ROTATE_LEFT) % POSSIBLE_ROTATIONS;
         updateRotation(rotation);
     }
+
+    /**
+     * Rotate the structure counter clockwise.
+     */
+    private void mirror()
```

Comment:
nope, I call it from a non-static context. But sonar doesn't recognize that type of methdo call still.